pkg/resource/db_cluster/custom_update.go

```diff
@@ -630,6 +608,16 @@ func (rm *resourceManager) newCustomUpdateRequestPayload(
 		}
 		res.SetVpcSecurityGroupIds(f23)
 	}
-
+        // For ServerlessV2ScalingConfiguration, MaxCapacity and MinCapacity,  both need appear in modify call to be modified
```

Comment:
I think it would actually be better if we separated them. Here is my logic:

If a user only has `MinCapacity` (and not `MaxCapacity`), then this code will silently ignore the scalingconfiguration change and not make the modify call. But everything else will appear to succeed. The controller will then be stuck in a loop as it describes a different `MinCapacity` to the one in the spec.

If we add them independently of the other, then if a user specifies only `MinCapacity`, the `ModifyDBCluster` call will fail. The error it returns will be populated into the condition of the object and it (hopefully) will be obvious to the user that they need to add `MaxCapacity`. Also, returning an error will pause reconciliation (with a `Terminal` condition) until it is updated.

Reply:
That's a good point! it makes sense to me